infra/modules/terraform-backend-s3/main.tf

```diff
@@ -269,10 +274,3 @@ resource "aws_s3_bucket_logging" "tf_state" {
   target_bucket = aws_s3_bucket.tf_log.id
   target_prefix = "logs/${aws_s3_bucket.tf_state.bucket}/"
 }
-
-resource "aws_s3_bucket_logging" "tf_log" {
-  bucket = aws_s3_bucket.tf_log.id
-
-  target_bucket = aws_s3_bucket.tf_log.id
-  target_prefix = "logs/${aws_s3_bucket.tf_log.bucket}/"
-}
```

Comment:
removed this for now because i'm running into a super cryptic error that only happens when running in GitHub actions (doesn't happen when i run CI locally)

![image](https://user-images.githubusercontent.com/447859/230695628-e707dc0e-95f6-447c-955e-456a8e714c6b.png)

see github run that failed: https://github.com/navapbc/template-infra/actions/runs/4641980887/jobs/8215462775